code/game/objects/structures/crates_lockers/closets/secure/vault.dm

```diff
@@ -30,12 +30,15 @@
 	name = "\improper Armory vault locker"
 	req_access = list(access_armory)
 	
-/obj/structure/closet/secure_closet/vault/armory/atoms_to_spawn()
-	if(Holiday == APRIL_FOOLS_DAY)
-		return list(
-			/obj/item/weapon/gun/projectile/rocketlauncher = 1,
-			/obj/item/ammo_casing/rocket_rpg = 3,
+/obj/structure/closet/secure_closet/vault/armory/atoms_to_spawn()`
+    return list(
```

Comment:
Put that part at the end of the proc, because else everything after the "return" will be ignored